keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -3,703 +3,644 @@
+```
+    BucketRequest - br1                                   BucketClass - bc1
+    |------------------------------|                      |----------------------------------|
+    | spec:                        |                      | spec:                            |
+    |   bucketClassName: bc1       |                      |   provisioner: s3.amazonaws.com  |
+    |   protocol: s3               |                      |   parameters:                    | 
+    |   deletionPolicy: delete     |                      |     key: value                   |
+    |------------------------------|                      |----------------------------------|
+```
 
-When the app pod terminates, the kubelet gRPC calls _NodeUnstageVolume_ and _NodeUnpublishVolume_, which the cosi-node-adapter also receives. The adapter orchestrates tear-down of the ephemeral volume provisioned for this bucket.
+###### 2. COSI creates a intermediate Bucket object
 
-General prep:
-+ admin creates the needed bucket classes and bucket access classes.
+The ControllerManager creates a Bucket object by copying fields over from both the BucketRequest and BucketClass. This step can only proceed if the BucketClass exists. This Bucket object is an intermediate object, with its status field `bucketAvailable` set to false. This indicates that the Bucket is yet to be created in the OSP.
 
-Prep for brownfield:
-+ admin creates `Bucket` instances representing backend buckets.
+More information about Bucket is [here](#bucket)
 
-## Create Bucket
+```
+    Bucket - br-$uuid
+    |--------------------------------------|
+    | name: br-$uuid                       |
+    | spec:                                |
+    |   bucketClassName: bc1               |
+    |   protocol: s3                       |
+    |   parameters:                        |
+    |     key: value                       |
+    |   provisioner: s3.amazonaws.com      | 
+    | status:                              |
+    |   bucketAvailable: false             |
+    |--------------------------------------|
+```
 
-![CreateBucket Workflow](images/create-bucket.png)
+###### 3. COSI calls driver to create Bucket
 
-This workflow describes the automation supporting creating a new (greenfield) backend bucket. Accessing this bucket is covered in [Sharing COSI Created Buckets](#sharing-cosi-created-buckets). COSI determines that a request is for an existing bucket when `BucketRequest.bucketInstanceName` is specified.
+The COSI sidecar, which runs alongside each of the drivers, listens for Bucket events. All but the sidecar with the specified provisioner will ignore the Bucket object. Only the appropriate sidecar will make a gRPC request to its driver to create a Bucket in the OSP.
```

Comment:
"... to create a Bucket in the OSP."  -> I think Bucket here should be lowercase, or there needs to be some way to distinguish the backend bucket from the COSI Bucket.